### ISSUE TITLE
Postman sort

### ISSUE THREAD

**Issue by jainaman224** [jainaman224/Algo_Ds_Notes#2251]
- [ ] C - @jhasaurabh312
- [x] C++ - @coderprasukj
- [x] C#
- [ ] CoffeeScript
- [ ] Dart - @Rukmini-Meda
- [ ] Go
- [ ] Java - @bluejay229
- [x] JS - @coderprasukj
- [ ] Kotlin - @myselfanuj
- [ ] Php
- [ ] Python - @jhasaurabh312
- [x] Readme - @shwetagurnani
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

**Comment by chaithanyarlk** [comment]
i would like to work on this in c++

**Comment by jhasaurabh312** [comment]
Hey i would like you to assign me it in c++ and java and readme. I have already sent a pr for c and python 3. Thanks !!!!

**Comment by srimani-programmer** [comment]
I want to work on it in Java. Please assign this issue to me @singh-shreya6 @raghav-dalmia @asha15

**Comment by AparnaJuhi** [comment]
I wanna work in C and C++.

**Comment by saloniankita23** [comment]
I want to work on this issue in C++. Kindly assign this issue to me.

**Comment by Ratna04priya** [comment]
I would like to do in Go, Java and rewrite the readme

**Comment by AJAYSHARMA2012** [comment]
Please assign this to me @jainaman224 . I can update the README

**Comment by SomyaGandhi** [comment]
I can work on this in javascript. Please assign this issue to me.

**Comment by bluejay229** [comment]
please assign this issue to me I can work on this in java.

**Comment by sheetalssr** [comment]
I CAN DO THIS JS .KINDLY ASSIGN ME THIS .

**Comment by urbashi123** [comment]
I would like to work on readme. Can you assign me please?
@jainaman224

**Comment by Kuljeet-123** [comment]
@jainaman224 I wanna work on this issue in PHP

**Comment by ritish099** [comment]
@jainaman224 I would like to submit it in C#. Can you assign it to me?

**Comment by Meenakshi2000** [comment]
please assign this one , can do in c++, java

**Comment by myselfanuj** [comment]
I would like to work on this algo for kotlin please assign me this issue for kotlin. @jainaman224

**Comment by ritish099** [comment]
@jainaman224 please review my 2 PR #2511 and #1952 . #1952 is pending from many days and it has one approval also

**Comment by shwetagurnani** [comment]
Please assign me readme

**Comment by prince-09** [comment]
I want to work on this in js @jainaman224

**Comment by chandrikadeb7** [comment]
I want to work on PHP.

**Comment by jainaman224** [comment]
@shwetagurnani @myselfanuj @bluejay229 @AparnaJuhi Are you working on it?

**Comment by myselfanuj** [comment]
Yes, I am working on it. I will give the PR ASAP.

**Comment by bluejay229** [comment]
Yes, I am working on it. I am sorry for the delay. I will give the PR soon.

**Comment by coderprasukj** [comment]
@jainaman224  I would like to work on this issue in JS. Please assign this issue to me.

**Comment by coderprasukj** [comment]
@jainaman224 I would like to work on this issue in C++. I have already did this issue in JS. Please Kindly assign it to me.

**Comment by Rukmini-Meda** [comment]
I would like to work on this issue in Dart. Please assign this to me as part of GSSOC'20.

**Comment by Tahanima** [comment]
Hi @jainaman224 @singh-shreya6 
I want to implement **Postman Sort** in Ruby. Please assign me this in Ruby.

**Comment by ritwikchakraborty123** [comment]
I would like to do it in PHP...
